docs/source/specification.md

```diff
@@ -72,13 +72,64 @@ Below we describe each level of the `rawdata` folder hierarchy in more detail. T
 
 ### Datatype
 
-The following datatypes are supported:
+The datatype folder, placed in the session level folder, is where
+data is stored. Two sets of datatype folder names are supported,
+either '*Broad*' or '*Narrow*'. The *Broad* datatype names are designed to
+cover most use cases:
 
 * `ephys`: electrophysiology (e.g. Neuropixel probes, tetrodes)
 * `behav`: behavioural (e.g. video and audio files, response logs)
 * `funcimg`: functional imaging (e.g. calcium and voltage imaging)
 * `anat`: anatomical (e.g. histology, using confocal or lightsheet)
 
+In some cases, the *Broad* datatype names may not be specific enough,
+for example if a session includes two different types of electrophysiological
+recordings, or anatomical imaging methods. In this case, the *Broad* datatype
+name can be substituted for a *Narrow* datatype name. See the dropdown below for
+the full list of supported *Narrow* datatype names.
+
+:::{dropdown} Narrow datatypes
+
+If a *Broad* datatype is replaced with a corresponding *Narrow* datatype,
+the *Broad* datatype should not be used.
```

Comment:
*must* not be used. We want this to be an absolute requirement I guess. If say "anat" and "spim" co-exits, all hell would break loose.